client/src/App.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react';
 
 import AppList from './components/AppList/AppList.jsx';
-import Login from './components/Login';
+import Login from './components/Login/Login';
 import Header from './components/Header';
-import Signup from './components/Signup';
+import Signup from './components/Signup/Signup';
```

Comment:
why duplicate paths?

Reply:
I created a folder for each login and signup jsx files to add scss files. Having all of them inside components folder without categorization seemed redundant.